src/scss/components/_checkbox.scss

```diff
@@ -0,0 +1,81 @@
+@use 'abstracts' as *;
+
+.modal {
+  fieldset {
+    margin: 0;
+    padding: 0;
+    border: 0;
+  }
+
+  .checkbox__container {
+    margin: 0.5rem 0;
+  }
+
+  input[type='checkbox'] {
+    position: absolute;
+    visibility: hidden;
+    opacity: 0;
+
+    + label {
+      position: relative;
+      display: block;
+      padding-left: 1.5rem;
+      font-family: $font__roboto;
+      cursor: pointer;
+
+      &::before {
```

Comment:
Add `transition: background 0.3s ease-in-out;` to this selector so the background smoothly transitions.